### PR TITLE
armors are now visible

### PR DESCRIPTION
**Pull request by francois76**
Now the armors are visible on other entities.

### CODE REVIEW

File: src/main/java/org/dragonet/proxy/network/translator/pc/PCEntityEquipmentPacketTranslator.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        if (entity == null) {
+            if (packet.getEntityId() == (int) session.getDataCache().get(CacheKey.PLAYER_EID)) {
+                entity = session.getEntityCache().getClientEntity();
+            } else {
+                return null;
+            }
+            return null;
+        }
+      
+    ItemStack items = packet.getItem();
```
Comment (HoverEpic) [inline src/main/java/org/dragonet/proxy/network/translator/pc/PCEntityEquipmentPacketTranslator.java:27]: There is a problem with your indentation from this point.

**Comment by lukeeey** [comment]
Ffs this file isnt even using TABs

HOW HARD IS IT

**Comment by francois76** [comment]
I've compared the indentation of the commited files with all the other ones, and I haven't seen any difference. I've seen that indentation is done with only space on each file, so I've replaced each tabs with 4 space in all my commits.

**Comment by DefinitlyEvil** [comment]
4 spaces = good,
tabs = bad 
personally speaking lel

**Comment by lukeeey** [comment]
Tabs are so much easier to work with. Spaces are for simpletons who would like to spend more time pushing SPACE

**Comment by francois76** [comment]
I always use tabs for my own projects, but lots of other classes, especially translators are indented with only spaces, so I follow the guidelines.

**Comment by lukeeey** [comment]
Half of the project is TABs, half is spaces and it gets annoying while coding when half certain lines of a file are spaces and the rest is tabs. Can everyone just use tabs

**Comment by HoverEpic** [comment]
I'm reformating classes when i view indentation problem, i'll make a full pass tomorrow.

**Comment by lukeeey** [comment]
There should be a button to reformat the entire project

**Comment by DefinitlyEvil** [comment]
Spaces pls no tabs

**Comment by lukeeey** [comment]
why not?

**Comment by DefinitlyEvil** [comment]
Tabs look bad in some editors.

**Review by HoverEpic — APPROVED** [review]

**Comment by yoft** [comment]
@lukeeey Most IDEs and editors allow the tab key to input four spaces, and in saying that, if your IDE struggles to handle indentation, it might be time to either do something about it, or stop complaining.

**Comment by lukeeey** [comment]
@yoft Or maybe its time to shut up because this conversation has already been resolved

**Pull request merged**